### ISSUE TITLE
Please release latest version in Cocoapods.

### ISSUE THREAD

**Issue by vigneshuvi** [magicalpanda/MagicalRecord#1335]
In [Cocoapods MagicalRecord](https://cocoapods.org/pods/MagicalRecord), We are unable to find the latest version. Can you please update into Cocoapods and let me know!

**Comment by Coeur** [comment]
It is the latest released version: 2.3.3 is only a Carthage fix as described by the release log: https://github.com/magicalpanda/MagicalRecord/releases

Although, I do admit I should merge a few pending pull requests!

**Comment by vigneshuvi** [comment]
Yeah understand from release note. Can you please maintain consistency version number in both Carthage and Cocoapods?

**Comment by Coeur** [comment]
I plan to make some updates to the repository during Chinese national holidays (first week of October).
Then I'll need to ask Tony or Saul for CocoaPods publication. So, be patient...

**Comment by Coeur** [comment]
OK, sorry for the delay, let's assume that I'm active again this week.
My goal is to understand and fix the `#BROKEN` from https://github.com/magicalpanda/MagicalRecord/blob/master/.travis.yml before making any new release.

**Comment by killerham** [comment]
@Coeur any update on this?

**Comment by Coeur** [comment]
Dear @killerham , thank you for asking.
The owners of the repository prefer to no longer make new releases for the pod 'MagicalRecord'. So basically people should stick to a direct reference to the branch they want. I'll try to update the readme to reflect that.

People are welcome to fork the project.

**Comment by Coeur** [comment]
@killerham readme updated: #1374.
I've assigned the issue to the [pod](https://cocoapods.org/pods/MagicalRecord) maintainers in case they change their mind one day (doubtful).

**Comment by tonyarnold** [comment]
Yeah, sorry folks - I'm not comfortable releasing something that I haven't tested, and I don't use MR in anything anymore. 

It's @casademora's baby: perhaps ask nicely if you can have `pod trunk` access for MR?

**Comment by magicalpandatest** [comment]
I did try to load up cocoa pods to update the version or add people to the list but I was unsuccessful. I don’t mind publishing a new version but I don’t want to make a mess with it either at this point

**Comment by vitaliy-solarwinds** [comment]
@magicalpandatest it's possible to restore access using https://trunk.cocoapods.org/claims/new form
